src/pocketmine/entity/Living.php

```diff
@@ -232,6 +232,14 @@ public function getEffect(int $effectId){
 	public function hasEffect(int $effectId) : bool{
 		return isset($this->effects[$effectId]);
 	}
+	
+	/**
+	 * Returns whether the any effect is active on the mob.
+	 * @return bool
+	 */
+	public function hasEffects() : bool{
+		return count($this->effects) > 0;
```

Comment:
This could also be `!empty($this->effects)`

Reply:
Yeah, I know, but what's better? To do as you have?